tools/frr-reload.py

```diff
@@ -1054,6 +1054,19 @@ def compare_context_objects(newconf, running):
                 for line in running_ctx.lines:
                     lines_to_del.append((running_ctx_keys, line))
 
+            # Some commands can happen at higher counts that make
+            # doing vtysh -c inefficient (and can time out.)  For
+            # these commands, instead of adding them to lines_to_del,
+            # add the "no " version to lines_to_add.
+            elif (running_ctx_keys[0].startswith('ip route') or
```

Comment:
would it help for `bgp as-path/community-list/extcommunity-list/large-community-list` maybe as well?

Reply:
Possibly, but I was limiting my changes to the ones I knew were an issue.  I can explore these others as well if needed.  Are you aware if deployments typically have large numbers of those items that might be deleted at the same time?

Reply:
One additional factor I didn't really spell out.   We are only hitting issues with over 100 items (routes, prefix-lists, access-lists) on super slow platforms.  In this case, i'm doing my testing an ARM boxes and can't make it fail on anything with more horsepower than that.

Reply:
I know huge deployments but they for sure aren’t on slow mobile phones 😀